releasenotes/notes/1.0/prepare-1.0-e13b9b8273055775.yaml

```diff
@@ -0,0 +1,71 @@
+---
+prelude: |
+  We are very pleased to release Qiskit 1.0.0. This release is the
+  culmination of 7 years of development to mature Qiskit into a stable, high-
+  performance SDK for quantum computing, and is the start of a new
+  era for the Qiskit project and community. Besides general performance and
+  stability improvements, the most significant changes with the 1.0.0 
+  release relate to the stability policy, release cycles, and versioning.
+  Qiskit is now formally using `semantic versioning <https://semver.org/>`__ ,
+  and this means that, for the entire lifecycle of the 1.x release series, the project
+  is commited to maintaining backwards compatibility. We're also now starting
+  to offer bugfix support for major versions, so that you continue to have a
+  supported branch for 6 months after the release of the next major version.
+  This starts now with the 1.x and 0.x major series: the 0.46.x release will
+  continue to be supported, with periodic patch releases that contain bug fixes, for 6
+  months after the release of 1.0. That is, 0.46.x will be supported until 2024-08. 
+  You can see the full details of the new policy here:
+
+  https://docs.quantum.ibm.com/start/install#qiskit-versioning
+
+  Also of key importance in this release is the change in packaging. Since the
+  Qiskit 0.7 release in 2018-12, when Qiskit introduced its
+  `elements <https://medium.com/qiskit/qiskit-and-its-fundamental-elements-bcd7ead80492>`__
+  model,  the ``qiskit`` package you would install has been a meta-package (or package of
+  packages) that installs the combined elements comprising Qiskit. As announced
+  in previous releases <link to 0.44, 0.45, and blog post>,  with the 1.0 release we're
```

Comment:
Don't forget to put this link in!

Reply:
Done in: https://github.com/Qiskit/qiskit/pull/11760/commits/f3c965e9b4338cb6ad0b1151eaf25effd9f7dd2c